x/cdp/abci.go

```diff
@@ -36,6 +39,14 @@ func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k keeper.Keeper)
 			panic(err)
 		}
 
+		if blockHeight%params.BeginBlockerExecutionBlockInterval != 0 {
+			ctx.Logger().Debug("skipping x/cdp SynchronizeInterestForRiskyCDPs and LiquidateCdps")
```

Comment:
Could throw `cp.Type` in these messages since they will be repeated for each type